cmd/server/server.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2021 Andy Dalton
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"net"
+	"os/signal"
+	"syscall"
+
+	"github.com/adalton/teleport-exercise/certs"
+	"github.com/adalton/teleport-exercise/pkg/command"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	argAddress string
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "jobmanager",
+	Short: "Run the job manager server",
+	Run: func(cmd *cobra.Command, args []string) {
+		runServer()
```

Comment:
Pass the command context here and remove the `context.Background()` call in `runServer`
```suggestion
		runServer(cmd.Context())
```